app/src/main/java/com/example/warehousemanagement/ui/common/TextField.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.warehousemanagement.ui.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.warehousemanagement.R
+import com.example.warehousemanagement.ui.theme.WarehouseManagementTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TextField(name: String) {
+    var productName by remember { mutableStateOf(TextFieldValue("")) }
```

Comment:
- Rename 
- this should be "rembersaveble"
- you have to pass this value in constructor . Then is 's available to get this string ( do not have Outlinetext.value)